docs/builders/index.mdx

```diff
@@ -26,3 +26,19 @@ the following VMware builders:
   an existing VMware VM you want to use as the source. As an additional
   benefit, you can feed the artifact of this builder back into Packer to
   iterate on a machine.
+
+## How to use this plugin
+
+From Packer v1.7.0, copy and paste this code into your Packer configuration to install this plugin.
+Then, run [`packer init`](/docs/commands/init).
+
+```hcl
+packer {
+  required_plugins {
+    vmware = {
+      version = ">= 0.0.1"
```

Comment:
Good catch. Let's set the minimum version so that folks reading the docs on Packer.io can get a sense of the version the documentation pertains to. 

```suggestion
      version = ">= 1.0.3"
```

Reply:
Thanks for the suggestion and merge.

Ryan